### DISCUSSION TITLE
VenusOS

### DISCUSSION THREAD

**Discussion by marc115uk** [asylumexp/Proxmox#89] [General]
Would like to be able to add VenusOS to proxmox on my pi. Is there any chance of getting a helper script for this? Thanks 🙏

**Comment by asylumexp** [comment]
> Would like to be able to add VenusOS to proxmox on my pi. Is there any chance of getting a helper script for this? Thanks 🙏

should ask upstream for this https://github.com/community-scripts/proxmoxve, i only fix scripts so they run on arm64, and if it does get made i wont be able to get it fixed until mid december but im not sure how quickly theyll add a script or if they even will.

**Reply by asylumexp** [reply]
https://github.com/community-scripts/ProxmoxVE/discussions/42

already a discussion for it
